tests/ha/hawk_gui.pm

```diff
@@ -92,60 +43,39 @@ sub run {
     }
 
     select_console 'root-console';
+    install_docker;
 
-    my $python = install_required_python_pkgs;
-    install_geckodriver $python;
-    # The line below can be changed for a call to lib/selenium.pm::enable_selenium_port()
-    # once Selenium::Remote::Driver it's available in the openqa.suse.de workers
-    assert_script_run('systemctl stop ' . opensusebasetest::firewall());
+    # TODO: Use another namespace in DockerHub
+    my $docker_image = "ricardobranco/hawk_test";
```

Comment:
Please don't use a "physical" name here, more the name of the team.

Reply:
A namespace for our team must be created.

Reply:
Ok, can you check this with Haris and Paul please?

Reply:
I was told to avoid DockerHub, so I have to use the SUSE Registry.  For this I have to use OBS.

To use OBS I have to:
1. Use an opensuse/leap as base image (which adds half a gig to the final Docker image). 
2. Connect the build with Gitlab.

Will do that.  Will try first if we still have space in the openQA VM.